ui/src/org/safe.ts

```diff
@@ -28,6 +28,36 @@ export async function getPendingTransactions(
   return response.results || [];
 }
 
+export interface MemberResponse {
+  threshold: number;
+  members: string[];
+}
+
+export async function getMetadata(
+  ethEnv: Ethereum.Environment,
+  safeAddress: string
+): Promise<MemberResponse> {
+  safeAddress = ethers.utils.getAddress(safeAddress);
+  const safeServiceClient = createSafeServiceClient(ethEnv);
+  const response = await safeServiceClient.getSafeInfo(safeAddress);
+
+  return <MemberResponse>{
```

Comment:
The return statement also checks without the explicit typecast. I would prefer to remove the explicit typecast because the it accepts excess properties
```typescript
return <MemberResponse> {
    foo: true,
    threshold: response.threshold,
    members: response.owners,
  };
```
Without the type cast the excess property produces an error.

Reply:
6f609f6943e714b73f452722231eff6caf14c608